src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -942,7 +944,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
             .collect { techSpec ->
                 [
                     key        : techSpec.key,
-                    req_key    : techSpec.requirements.join(", "),
+                    req_key    : techSpec.requirements.join(", ")?:"None",
```

Comment:
@martsec what do you intend to do here? If you want to check for null, you'll have to be explicit and say: `techSpec.requirements ? techSpec.requirementss.join(", ") : "None". They way you do it here will not likely work, because `join` will throw a NPE, not return a null value.

Reply:
```
println([]? [].join(","):"nuuuuull") // output: nuuuuull
println([].join(",")?:"nuuuuull")​​​​​​​​​​​​​ // output: nuuuuull
```

Those two pieces of code behave the same way. The second one is more concise.